### PR TITLE
Reorganize server files and database connections

### PR DESCRIPTION
**Pull request by pafrias**
Server index has been moved to root file, server functions have
been separated into their appropriate files, and all database
connections refer to a single config file.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,21 +1,3 @@
...
-          "node": true
-    },
-    "plugins": [
-        "react"
-     ],
-     "rules": {
-       "react/prop-types": 0,
-       "react/destructuring-assignment": 0
-     }
+    "extends": "airbnb"
```
Comment (pafrias) [inline .eslintrc.js:2]: This part may look odd; I noticed that the source code was not listing style errors as I expect, so I just deleted the original file and re-initialized eslint.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const compress = require('compression');
+const { readComments, readCount } = require('./app/routers/router.js');
```
Comment (JackyLei94) [inline index.js]: I might be wrong, but I think this is the controllers and not the router. Let's talk about it.

**Review by JackyLei94 — CHANGES_REQUESTED** [review]
See comment about router vs. controller

File: index.js (hunk 2 above)
Comment (pafrias) [reply index.js]: Are you requesting a change of file name, dir name, or both?

**Pull request closed without merging**